reflect.go

```diff
@@ -120,8 +121,54 @@ func runInDir(program []byte, dir string) (*model.PackedPkg, error) {
 		return nil, err
 	}
 
+	{
+		// Copy go.mod into the build directory:
+		wd, err := os.Getwd()
+		if err != nil {
+			log.Fatalf("Unable to load current directory: %v", err)
+		}
+
+		modRoot := findModuleRoot(wd)
+
+		if modRoot != "" {
+			MustCopyFile(filepath.Join(modRoot, "go.mod"), filepath.Join(tmpDir, "go.mod"))
+		}
+
```

Comment:
Could you remove this newline? Is this autoformatted?

Reply:
> Could you remove this newline?

Removed.

> Is this autoformatted?

Yes, it is. The standard Go formatter is completely fine with newlines in positions like that. For a stricter formatter, see https://github.com/mvdan/gofumpt

Reply:
As long as it's autoformatted I'm not particularly fussed.